src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1556,3 +1556,13 @@ def progress_callback(acl_changes):
         failed_entries.append(acl_changes.batch_failures)
 
     namespace.progress_hook = progress_callback
+
+
+def get_not_none_validator(attribute_name):
+    def validate_not_none(cmd, namespace):
+        attribute = getattr(namespace, attribute_name, None)
+        options_list = cmd.arguments[attribute_name].type.settings.get('options_list')
+        if not attribute:
```

Comment:
```suggestion
        if attribute is None:
```

consider  attribute value is 0 or False

Reply:
Good catch.
Changed it to `if attribute in (None, '')` since I want to allow 0 or False but not allow empty string.